router.go

```diff
@@ -201,44 +211,63 @@ func (app *App) next(c *DefaultCtx) (bool, error) {
 	return false, err
 }
 
-func (app *App) requestHandler(rctx *fasthttp.RequestCtx) {
-	// Handler for default ctxs
-	var c CustomCtx
-	var ok bool
-	if app.newCtxFunc != nil {
-		c, ok = app.AcquireCtx(rctx).(CustomCtx)
-		if !ok {
-			panic(errors.New("requestHandler: failed to type-assert to CustomCtx"))
-		}
-	} else {
-		c, ok = app.AcquireCtx(rctx).(*DefaultCtx)
-		if !ok {
-			panic(errors.New("requestHandler: failed to type-assert to *DefaultCtx"))
+func (app *App) defaultRequestHandler(rctx *fasthttp.RequestCtx) {
+	// Acquire DefaultCtx from the pool
+	ctx, ok := app.AcquireCtx(rctx).(*DefaultCtx)
+	if !ok {
+		panic(errors.New("requestHandler: failed to type-assert to *DefaultCtx"))
+	}
+
+	defer app.ReleaseCtx(ctx)
+
+	// Check if the HTTP method is valid
+	if ctx.methodINT == -1 {
```

Comment:
I may have introduced a bug here, I need to add a check to replace `app.methodInt()`

Reply:
Tested locally, it's fine.